github-changelog-generator.php

```diff
@@ -1,242 +1,113 @@
 <?php
 
+// "autoload" our classes
+if (!class_exists('ins0\GitHub\Repository')) {
+    require_once 'src/Repository.php';
```

Comment:
Completely forgot about that. It's up to you whether or not I should break compat. Let me know and I'll rewrite the diff.